test/handler.test.js

```diff
@@ -472,6 +498,36 @@ describe('strong-error-handler', function() {
           done);
     });
 
+    it('sanitizes response to prevent XSS', function(done) {
+      var error = new ErrorWithProps({
+        name: 'Error<img onerror=alert(1) src=a>',
+        message:
+          'No instance with id <img onerror=alert(1) src=a> found for Model',
+        statusCode: 404,
+        details: 'some details<img onerror=alert(1) src=a>',
+        extra: 'sensitive data<img onerror=alert(1) src=a>',
+      });
+      givenErrorHandlerForError(error, {debug: false});
+      requestHTML()
+        .end(function(err, res) {
+          expect(res.statusCode).to.eql(404);
+          var body = res.error.text;
+          expect(body).to.match(
+            /some details&lt;img onerror=alert\(1\) src=a&gt;/
+          );
+          expect(body).to.not.match(
```

Comment:
Please remove these `not.match` assertions. There are other tests that have already verified that sensitive information is excluded. Let's keep this new test focused on HTML escaping only. Please remove `details` and `extra` from the error object created at the beginning of this test too, since they won't be used anymore.